src/protocol.cpp

```diff
@@ -128,12 +128,20 @@ bool CMessageHeader::IsValid(const MessageStartChars& pchMessageStartIn) const
     return true;
 }
 
+// NOTE: After we fork, we can require these service bits
```

Comment:
we just need to pick a fork height for this? (or we could tie it in with the upcoming hash computation fork -- PR #209 ).